### PR TITLE
shorten or redirect the reachout section

### PR DESCRIPTION
**Pull request by CaptainSifff**
Originally we had the reach-out section. I can still find arguments, but it had to be redirected in order to make sense in the srtucture of this paper. Here's my try at that.....

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -1037,18 +1037,13 @@ They are able to communicate and collaborate with lawyers.
...
-This transition usually happens slowly.
-A typical RSE development path starts off with a PhD student programming a tool that others notice and find useful.
-Soon the proto RSE is known to have programming skills and is sought after to join other projects.
-The researcher is now well on the way to becoming an RSE.
-If they enjoy this role and would like to take it further they need to know that there is an RSE career path as well as that specialised training materials exist.
-Beyond that, for an RSE it is important to be a part of a network with other RSEs, irrespective of the career level.
+# Connecting RSEs {#sec:reachout}
+RSEs gather around a uniform set of values and competencies and thereby 
+naturally form networks.
+For an RSE it is important to be a part of a network with other RSEs, irrespective of the career level in order to be more
```
Comment (MakisH) [inline competencies.md:1043]:
```suggestion
For an RSE, it is important to be part of a network with other RSEs, regardless of career level, in order to be more
```

File: competencies.md [hunk 2]
```diff
@@ -1037,18 +1037,13 @@ They are able to communicate and collaborate with lawyers.
...
+effective at supporting research.
 A perfect first step for forming this network is topic-related conferences, workshops or meet-ups.
```
Comment (MakisH) [inline competencies.md:1045]:
```suggestion
A perfect first step for forming this network is topic-related conferences, workshops, or meet-ups.
```

File: competencies.md [hunk 3]
```diff
@@ -1037,18 +1037,13 @@ They are able to communicate and collaborate with lawyers.
...
 Beyond that, there is the broader RSE community organised at the local and regional level with chapters or local/regional communities, at the national level with societies and the international RSE society.
-Each of them offers possibilities for connecting within or beyond an individual institution and is a great way to find like-minded people to grow a wider network and thereby facilitate the sharing of information on interesting events or help each other out.
 This available layered network can greatly benefit the RSE in finding help with issues outside of their own comfort zone
 and provides a welcoming, social safety net providing a home for the RSE. Since we feel providing aspiring RSEs with this net
 is of utmost importance we envision compulsory events introducing that to young RSEs.
```
Comment (MakisH) [inline competencies.md:1049]:
```suggestion
is of utmost importance, we envision compulsory events introducing that to young RSEs.
```

**Review by MakisH — COMMENTED** [review]
Some minor language fixes.

Overall, I am not really sure how this section fits in this paper. I think the main message is that there are RSE-related networks and events. Is this really worth it a complete (tiny) section, or could be just 1-2 sentences in the introduction?

My gut feeling tells me the paper will be as good or even better without this section, without missing any key message.

File: competencies.md (hunk 3 above)
Comment (MakisH) [inline competencies.md:1049]: I am not sure I understand or agree with the "we envision compulsory events introducing that to young RSEs":

1. Nothing else in this paper is described as "compulsory"
2. Who likes being told "you have to join this event"? I think motivation is something one should definitely bring, but compulsory events are strange.
3. I would see this similarly to any professional joining conferences and similar events common in their profession. Continuous learning (e.g., via networking) is a trait of every professional that wants to keep doing a good job.

Comment (CaptainSifff) [reply competencies.md]: The idea here was more that already in their masters courses available RSE networks are introduced. I don't know about your studies, but I think most people are just left at the end of their master's degree, when they are basically thrown out of university. The idea here is to emphasize the social aspect that is also in the communication skills and to integrate graduating RSEs as soon as possible into the existing networks.

Comment (MakisH) [reply competencies.md]: I studied CSE and we were also just left alone to continue. The same I think holds for every studies, but I think the focus here is the profession, not the studies.

I still think that it is kind of a common problem in almost every profession, and that this section does not really add value.

What would make more sense would be to shift the focus to lifelong learning: After I graduate from M.Sc. RSE and I also specialize into something, and I start working in team projects, how can I keep learning?

Even just renaming this section to `Lifelong learning` would improve how this section fits to the rest of the paper.

Comment (mhagdorn) [reply competencies.md]: "compulsory events" does sound very strong. Maybe tune it down a little and say

"we envision that young RSEs are alerted to these events"

Comment (mhagdorn) [reply competencies.md]: I am happy with @MakisH suggestion to shorten this to a few sentences and move them to the intro. I think what is described here holds for all graduates of any degree.

File: competencies.md (hunk 2 above)
Comment (jpthiele) [reply competencies.md:1045]: Also not sure if it shouldn't be 'are' instead of 'is' as we are then starting with a list. 
At least that's my gut feeling, but it might be wrong.
@jcohen02 any thoughts on this?

**Review by jpthiele — APPROVED** [review]

File: competencies.md (hunk 2 above)
Comment (jcohen02) [reply competencies.md:1045]: I think "is" is correct here @jpthiele since the subject the "first step" is singular.

File: competencies.md (hunk 3 above)
Comment (CaptainSifff) [reply competencies.md]: OK, since @MakisH succesfully shortened the intro to the length it now has, I will not add onto that.
The networking aspect is in 4.4, Tasks and Responsibilities already. Therefore it will be integrated there, mostly

**Comment by CaptainSifff** [comment]
On general consent, I removed the section. I redistributed parts to the responsibilities and to the future work section. A homework would be to make sure that at least the notion of providing networks AND embedding junior RSEs is reflected in the tables of #184 .

File: competencies.md [hunk 4]
```diff
@@ -533,9 +533,11 @@ what they would like to have learnt. Among the top five things mentioned were:
   knowledge of RC, USERS, and also NEW, in order to effectively interact with
   domain scientists. Today, an example is a presence on social media. The other
   TEAM-related aspect is the embedding of RSE graduates into the community of
-  RSEs. We envision our RSE graduates to be a part of a strong network of other
-  RSEs, tool-related communities and the classical domain communities. This point
-  is further elaborated in @sec:reachout.
+  RSEs, sharing the same set of values and competencies. We envision our RSE graduates to be a part of a strong network of other
+  RSEs, tool-related communities and the classical domain communities, making them more effective at supporting research.
```
Comment (MakisH) [inline competencies.md:537]:
```suggestion
  RSEs, tool-related communities, and the classical domain communities, making them more effective at supporting research.
```

File: competencies.md [hunk 5]
```diff
@@ -1084,7 +1065,11 @@ In that publication, we will detail how we organise our institutions and what qu
 need to have in order to effectively communicate our values.
 We will put forward ideas on how to build up bachelor's and master's programmes,
 of which a glimpse can already be found in this paper's appendix.
-We will show how we intend to provide the necessary continuous education for RSEs after graduation.
+We will show how we intend to provide the necessary continuous education for RSEs after graduation
+and we will connect that with the integration of RSEs into a mesh of community networks aimed at supporting research
```
Comment (MakisH) [inline competencies.md:1069]:
```suggestion
and we will connect that with the integration of RSEs into a mesh of community networks aimed at supporting research,
```

**Review by MakisH — APPROVED** [review]
Ah, much nicer now! :+1: :rocket:

**Review by jpthiele — APPROVED** [review]
